### PR TITLE
harmonoid: Update to version 0.3.6

### PR DESCRIPTION
**Pull request by Zliced13**
Relates to #10625

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Zliced13** [comment]
@rashil2000

File: bucket/harmonoid.json [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    "notes": [
+        "To install harmonoid file associations, run:",
+        "install-harmonoid-associations.ps1",
+        "To install the harmonoid context menu, run:",
+        "install-harmonoid-context.ps1",
+        "",
+        "To uninstall harmonoid file associations, run:",
+        "uninstall-harmonoid-associations.ps1",
+        "To uninstall the harmonoid context menu, run:",
+        "uninstall-harmonoid-context.ps1"
```
Comment (rashil2000) [inline bucket/harmonoid.json:18]:
```suggestion
        "To install harmonoid file associations, run:",
        "$dir\\install-harmonoid-associations.ps1",
        "To install the harmonoid context menu, run:",
        "$dir\\install-harmonoid-context.ps1",
        "",
        "To uninstall harmonoid file associations, run:",
        "$dir\\uninstall-harmonoid-associations.ps1",
        "To uninstall the harmonoid context menu, run:",
        "$dir\\uninstall-harmonoid-context.ps1"
```

File: bucket/harmonoid.json [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        "    Set-Content \"$dir\\$($_.Name)\" $content -Encoding 'utf8' -Force",
+        "}"
+    ],
+    "bin": [
+        "harmonoid.exe",
+        "install-harmonoid-associations.ps1",
+        "uninstall-harmonoid-associations.ps1",
+        "install-harmonoid-context.ps1",
+        "uninstall-harmonoid-context.ps1"
+    ],
```
Comment (rashil2000) [inline bucket/harmonoid.json:43]:
```suggestion
    "bin": "harmonoid.exe",
```

File: bucket/harmonoid.json [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        [
+            "harmonoid.exe",
+            "Harmonoid"
+        ]
+    ],
+    "pre_uninstall": [
+        "if ($cmd -eq 'uninstall') {",
+        "   Remove-Item 'HKCU:\\Software\\Classes\\Directory\\shell\\HarmonoidAddToPlaylist' -ErrorAction 'SilentlyContinue' -Recurse -Force",
+        "",
+        "   $formats = @(",
```
Comment (rashil2000) [inline bucket/harmonoid.json:54]: Please put the formats in a single line

**Review by rashil2000 — CHANGES_REQUESTED** [review]

File: bucket/harmonoid.json (hunk 2 above)
Comment (Zliced13) [reply bucket/harmonoid.json:43]: Why would I remove that from the bin? It doesn't make sense why would I want to take away the convenience away from the user?

Comment (rashil2000) [reply bucket/harmonoid.json:43]: Installing and removing associations is a one-time procedure - no need to pollute the shims directory for that. The `notes` section has also been updated above to use the full path of the script.

**Pull request merged**